src/main/scala/com/lucidworks/spark/util/SolrRelationUtil.scala

```diff
@@ -238,29 +238,29 @@ object SolrRelationUtil extends LazyLogging {
   }
 
   def applyFilter(filter: Filter, solrQuery: SolrQuery, baseSchema: StructType): Unit = {
-   filter match {
-     case f: And =>
-       val values = getAllFilterValues(f, baseSchema, ListBuffer.empty[String])
-       values.foreach(v => solrQuery.addFilterQuery(v))
-     case f: Or =>
-       val values = getAllFilterValues(f, baseSchema, ListBuffer.empty[String])
-       val fqStringBuilder = new StringBuilder
-       for (i <- values.indices) {
-         if (i == 0) fqStringBuilder.append("(")
-         fqStringBuilder.append(values(i))
-         if (i != values.size-1) {
-           fqStringBuilder.append(" OR ")
-         } else {
-           fqStringBuilder.append(")")
-         }
-       }
-       if (fqStringBuilder.nonEmpty) solrQuery.addFilterQuery(fqStringBuilder.toString())
-     case f: Not =>
-       f.child match {
-         case c : IsNull => solrQuery.addFilterQuery(fq(IsNotNull(c.attribute), baseSchema))
-         case _ => solrQuery.addFilterQuery("NOT " + fq(f.child, baseSchema))
-       }
-     case _ => solrQuery.addFilterQuery(fq(filter, baseSchema))
+    filter match {
```

Comment:
[0] Bleh.  Thanks for fixing this.  We should think about deciding on a code formatter for IntelliJ that enforces indenting, import optimization, etc. to cut down on some of the noise in these diffs.

Reply:
yeah, we should enforce code formatter all across our repos